app/scripts/controllers/landing.js

```diff
@@ -44,10 +45,26 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        $scope.filterUpdate();
       });
     }, 500, {leading: false});
 
-    $scope.dateFilter = function(event){
+    $scope.filterSearchResults = function(){
+      return _.filter($scope.searchResults, event => {
+        return dateFilter(event) && locationFilter(event);
+      });
+    };
+
+    $scope.filterUpdate = function(){
+      analytics.digitalData.searchTerm = $scope.searchVal;
+      analytics.digitalData.searchLocationFilter = $scope.eventFilters.locationName;
+      analytics.digitalData.searchDateFilter = $scope.eventFilters.date;
```

Comment:
@OzzieOrca I verified that the digitalData layer updates with the search term, location filter, and data filter when used. Seems like the pageview direct call rule fires each time something happens on ERT like a search. I'm thinking for the search interactions I could just listen for a change in the search data element. I would need to test that. 

Also, I'm not getting SSO Guid or pagenames. All pages names in the data layer are the same. See attachment for details. I did however get the logged in status to work.

<img width="960" alt="screen shot 2017-06-29 at 9 38 41 am" src="https://user-images.githubusercontent.com/20176352/27690678-02c6341c-5cb0-11e7-87c0-c44e23424f97.png">